### ISSUE TITLE
The Set default shell to bash? prompt is not asking on every connection to the system

### ISSUE THREAD

**Issue by beckhamrryyaann** [codefori/vscode-ibmi#2349]
Save us time! Click 'Report an Issue' from the Help and Support panel inside of VS Code after you have connected to a system.

**Describe the bug**
It appears that setting the default shell prompt only happens on the initial connection.  After sitting in a recent user group working with other developers that do not have alot of experience with the pase environment I think it would be good to ask the user every time if they want to set their default shell to bash, or almost force them!

**Comment by worksofliam** [comment]
@beckhamrryyaann Yes, I do think it is a good idea to use the SQL method as a fallback if `chsh` is not installed!

**Comment by sebjulliand** [comment]
> Great. I will try to get that change in. Would have been very helpful this past week. Trying to do a lab with most users using the ksh shell is pretty painful.

Ouch 😨

**Comment by SanjulaGanepola** [comment]
Closing as this was completed in https://github.com/codefori/vscode-ibmi/pull/3131
